### PR TITLE
POWR 232 Group content menu

### PR DESCRIPTION
**Pull request by kxwang**

### CODE REVIEW

File: web/sites/default/settings.php [hunk 1]
```diff
@@ -53,4 +53,6 @@
   if (is_array($settings)) {
     $settings['trusted_host_patterns'] = array('^'. preg_quote($primary_domain) .'$');
   }
-}
\ No newline at end of file
+}
+
+$settings['hash_salt'] = "testtest";
```
Comment (joshuami) [inline web/sites/default/settings.php]: @kxwang, you accidentally included a change to the settings.php. This value should be set in your settings.local.php to ensure that you don't need to update a file that can accidentally get pushed to production.

File: web/sites/default/services.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ parameters:
     #
     # Not recommended in production environments
     # @default false
-    debug: false
+    debug: true
```
Comment (joshuami) [inline web/sites/default/services.yml]: Similar to the settings.php change, you shouldn't need to update services.yml very often. This value should be set in your settings.local.php to use the development.services.yml file for your local environment. This ensures that we don't push a debugging setting all the way to prod.

**Comment by kxwang** [comment]
OK, I've reverted both accidental changes.

**Comment by joshuami** [comment]
I just pushed a rebase, so you can skip that. I need this code for my columns story POWR-90. I have a feeling it will introduce a lot of merge conflicts to resolve.

**Comment by kxwang** [comment]
Thanks, Josh!

**Pull request merged**